decompiler/backend/cexpressiongenerator.py

```diff
@@ -361,3 +360,13 @@ def _format_string_literal(constant: expressions.Constant) -> str:
             escaped = string_representation.replace('"', '\\"')
             return f'"{escaped}"'
         return f"{constant}"
+
+    @staticmethod
+    def format_variables_declaration(var_type: Type, var_names: list[str]) -> str:
+        """ Return a string representation of variable declarations."""
+        match var_type:
+            case Pointer(type=FunctionTypeDef() as fun_type):
+                rest = ", ".join(map(lambda n: f"(* {n})({', '.join(str(x) for x in fun_type.parameters)})", var_names))
```

Comment:
- Could you please use more meaningful name than `rest`?
- One-liner here is hard to understand. Could you please come up with more readable code here?

Reply:
Sure, I've split the one-liner up into multiple lines with more descriptive variable names.